.yamato/code-coverage.yml

```diff
@@ -0,0 +1,21 @@
+{% metadata_file .yamato/project.metafile %}
+---
+{% for project in projects -%}
+code_coverage_win_{{ project.name }}:
+  name: Code Coverage Report - Windows - {{ project.name }}
+  agent:
+    type: Unity::VM
+    image: package-ci/win10:stable
+    flavor: b1.large
+  commands:
+    - pip install unity-downloader-cli --upgrade --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+    - unity-downloader-cli -u trunk -c editor --wait --fast
```

Comment:
A lot of the other "one version" jobs just use `{{ test_editors.first }}` to keep all the editor configuration in one place. I don't feel strongly one way or the other, but might be a bit odd in the future to have coverage run against a different version.

Reply:
Plan is to run this on trunk to start out with to ensure we're liking it. If it's good, we can expand to other versions/branches